src/diffusers/pipelines/flux/pipeline_flux.py

```diff
@@ -429,6 +435,26 @@ def check_inputs(
         if max_sequence_length is not None and max_sequence_length > 512:
             raise ValueError(f"`max_sequence_length` cannot be greater than 512 but is {max_sequence_length}")
 
+        if latents is not None:
+            if not isinstance(latents, torch.Tensor):
+                raise ValueError(f"`latents` has to be of type `torch.Tensor` but is {type(latents)}")
+
+            if not _are_latents_packed(latents):
```

Comment:
But later in `prepare_latents()`, we are packing the latents and throwing a warning, no? So, I think we can remove this check.

Reply:
Same, this check can be removed from the `check_inputs` function.